org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/XMLFormatterTest.java

```diff
@@ -2340,6 +2340,18 @@ public void dontEnforceSingleQuoteStyle() throws BadLocationException {
 		format(content, expected, settings);
 	}
 
+	@Test

```

Comment:
Could you add 2 tests with:

 * single + enforce
 * double + enforce

Reply:
fixed